src/docs/page-components.jsx

```diff
@@ -4,36 +4,24 @@ import Avatars from '../components/avatar/pages/avatar-interactive';
 import Bubbles from '../components/bubble/pages/bubble-interactive';
 import Badges from '../components/badges/pages/badges-interactive';
 import Buttons from '../components/buttons/pages/buttons-interactive';
+import IconsAsButtons from '../components/icon-as-button/pages/icon-as-button-interactive';
+import slugify from 'slugify';
```

Comment:
good catch, although I just spotted that we can't use `slugify` it's just too big. It increases bundle size 4 times. I'll find something smaller.